client/src/Components/Input/Input.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { InputComponent } from './InputStyles';
+
+export type InputProps = {
+  type: string;
+  placeholder?: string;
+  inputValue?: string;
+  onChange?: () => void;
+  isRequired?: boolean;
+} & React.InputHTMLAttributes<HTMLInputElement>;
+
+const Input = ({ type, placeholder, inputValue, onChange, isRequired = false, ...otherProps }: InputProps) => {
+  return (
+    <label>
```

Comment:
Hmmm... Na razie to wygląda dziwnie, możemy albo to usunąć albo dodajkmy do tego komponentu propsa w którym byśmy przekazywali wartość w label, może być default pusty